project/assets/common.less

```diff
@@ -55,6 +56,7 @@ a {
 .navigation-bar-skin {
```

Comment:
less allows you to build definitions like this:
`.navigation-bar {
    &-skin {definitions;}
    &-active {definitions}
}`
This way the code gets a bit of structure